docs/OMS-Agent-for-Linux.md

```diff
@@ -380,7 +380,31 @@ If you are using selinux, the semanage tool can be used to allow TCP traffic for
 	  tag oms.syslog
 </source>
 ```
-**Note:** ports 0-1024 are privileged, and require special permissions. As the OMS Agent for Linux runs as the `omsagent` user trying to allocate a source port of 0 to 1024 results in agent startup failure
+### Syslog troubleshooting
+**Note:** ports 0-1024 are privileged, and require special permissions. As the OMS Agent for Linux runs as the `omsagent` user trying to allocate a source port of 0 to 1024 results in agent startup failure.
+
+**Checking for existance of a syslog collector**
+
+Make sure there is a syslog collector available by running the following command in shell:
+```
+	[ -f /etc/rsyslog.conf ] && echo "rsyslog is configured"
+	[ -f /etc/syslog-ng/syslog-ng.conf ] && echo "syslog-ng is configured"
+```
+
+**Checking port bindings using netstat**
+```
+	PORT=<port you use>
+	netstat -an | grep $PORT
+```
+**Checking port availability using netstat**
+```
+	PORT=<port you want to use>	
+	[ -z "`netstat -an | grep $PORT`" ] && echo "$PORT port is available" || echo "$PORT port is NOT available"
+```
+**Customizing syslog log format using fluentd' format regexp parameter if syslog data shows up trancated in OMS portal**
```

Comment:
typo: "using **FluentD's** format"... with s and capitalization
mispell: truncated